### ISSUE TITLE
resource citrixadc_servicegroup_servicegroupmember_binding doesn't support import

### ISSUE THREAD

**Issue by ewatt** [citrix/terraform-provider-citrixadc#233]
Import support can be added with the following patch

```
diff --git a/citrixadc/resource_citrixadc_servicegroup_servicegroupmember_binding.go b/citrixadc/resource_citrixadc_servicegroup_servicegroupmember_binding.go
index 086131a6..afadc55e 100644
--- a/citrixadc/resource_citrixadc_servicegroup_servicegroupmember_binding.go
+++ b/citrixadc/resource_citrixadc_servicegroup_servicegroupmember_binding.go
@@ -18,6 +18,9 @@ func resourceCitrixAdcServicegroup_servicegroupmember_binding() *schema.Resource
                Create:        createServicegroup_servicegroupmember_bindingFunc,
                Read:          readServicegroup_servicegroupmember_bindingFunc,
                Delete:        deleteServicegroup_servicegroupmember_bindingFunc,
+               Importer: &schema.ResourceImporter{
+                       State: schema.ImportStatePassthrough,
+               },
                Schema: map[string]*schema.Schema{
                        "customserverid": &schema.Schema{
                                Type:     schema.TypeString,
```

Am I able to submit a pull request?

**Comment by vadharm** [comment]
Thanks @ewatt , we are not accepting pull request as of now. But will review this internally, and have this enhancement soon. @giorgos-nikolopoulos  -Lets plan for this.

**Comment by giorgos-nikolopoulos** [comment]
Fixed with #271
